CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMake")
 
 project(cpr)
 
+set(CPR_MAJOR_VERSION 1)
+set(CPR_MINOR_VERSION 2)
+set(CPR_PATCH_VERSION 0)
+set(CPR_VERSION ${CPR_MAJOR_VERSION}.${CPR_MINOR_VERSION}.${CPR_PATCH_VERSION})
```

Comment:
The version can be specified in [project()](https://cmake.org/cmake/help/v3.0/command/project.html) without having to use 3 extra variables.

Example:

``` cmake
project(cpr
  VERSION 0.1.2
  LANGUAGES CXX)
```

This will also automatically generate the 3 variables `cpr_VERSION_MAJOR`, `cpr_VERSION_MINOR` and `cpr_VERSION_PATCH`.